spec/result_merger_spec.rb

```diff
@@ -73,14 +72,23 @@
               expect(SimpleCov::ResultMerger.results.length).to eq(1)
             end
           end
+        end
+      end
 
-          context "with merging disabled" do
-            before { SimpleCov.use_merging false }
+      describe ".store_result" do
+        it "refreshes the resultset" do
+          set = SimpleCov::ResultMerger.resultset
+          SimpleCov::ResultMerger.store_result({})
+          new_set = SimpleCov::ResultMerger.resultset
+          expect(new_set).not_to equal(set)
+        end
+      end
 
-            it "returns nil for SimpleCov.result" do
```

Comment:
This spec was misleading in that it only returned nil if `.running == false`, and it also happened independently of `use_merging` ... so it strikes me as not very useful, and a better spec can be found in spec/simplecov_spec.rb:22

Note that the behavior has now changed slightly; previously `.result` always returned `nil` if SimpleCov wasn't running. Now it only returns `nil` if it's not running **and** `use_merging == false`. If `use_merging == true`, you always get something useful from `result` (whether or not it merges the current `Coverage.result` depends on `.running`). This is useful for times when you have a post-CI script for formatting results and coverage thresholds, and the script itself doesn't need coverage (say, you have a script that aggregates and merges `.resultset.json` from many distributed workers, and want to format and enforce coverage on the final result).